packages/caravan-wallets/src/coldcard.ts

```diff
@@ -31,6 +31,11 @@ import {
 import { P2SH, P2SH_P2WSH, P2WSH } from "@caravan/bitcoin";
 import { BitcoinNetwork } from "@caravan/bitcoin";
 import { WalletConfigKeyDerivation } from "./types";
+// import {
+//   convertLegacyInput,
+//   convertLegacyOutput,
+//   getUnsignedMultisigPsbtV0,
+// } from "@caravan/psbt";
```

Comment:
Is this planned for this PR? If not, maybe a comment about what is going on with this commented-out code would be helpful.

Reply:
It kind of depends on capacity to finish the original goal of this PR and/or if we feel ok putting this in more or less as is. If we want to merge this in before fully enabling spend to taproot then I can do that. Otherwise, I put this TODO to make sure I know where to come back to.